src/qt/overviewpage.cpp

```diff
@@ -283,7 +283,7 @@ void OverviewPage::on_btnWebsite_pressed()
 
 void OverviewPage::on_btnBX_pressed()
 {
-    QDesktopServices::openUrl(QUrl("https://www.gridcoinstats.eu/block/"));
+    QDesktopServices::openUrl(QUrl("https://www.gridcoinstats.eu/block/#pk_campaign=GridcoinWallet&pk_kwd=" + QString::fromStdString(FormatFullVersion())));
```

Comment:
So many commits just for url change :D
You should have done this pull request yourself @startailcoon.